code/modules/cargo/exports/manifest.dm

```diff
@@ -4,6 +4,7 @@
 	cost = 200
 	k_elasticity = 0
 	unit_name = "approved manifest"
+	cost_modifiers = list("Nanotrasen", "Syndicate", "Solgov")	//everyone loves paperwork. Right?
```

Comment:
"yes hello here is an approved manifest of shit we bought from NanoTrasen" -Man's last word to syndicate officer